src/app/pages/invoices/list/invoices-table-data-source.ts

```diff
@@ -110,7 +119,8 @@ export class InvoicesTableDataSource extends TableDataSource<BillingInvoice> {
         sortable: true,
       },
     );
-    if (this.authorizationService.isAdmin()) {
+    // Columns only available for authorized users
+    if (this.invoicesAuthorizations.canListUsers) {
```

Comment:
> @LucasBrazi06 : I thought that columns are hidden automatically when they do not match the projected fields. Are we still supposed to check permissions when adding them to the list?

This is old code to remove.
You can check :-)

Reply:
@LucasBrazi06 @ClaudeROSSI I removed the check, and adapted what the backend returns so that it doesn't return user information for basic user => column will not show